src/Temporalio/Bridge/OptionsExtensions.cs

```diff
@@ -180,11 +184,7 @@ public static unsafe Interop.MetricsOptions ToInteropOptions(
                 {
                     throw new ArgumentException("Prometheus options must have bind address");
                 }
-                prometheus = scope.Pointer(
-                    new Interop.PrometheusOptions()
-                    {
-                        bind_address = scope.ByteArray(options.Prometheus.BindAddress),
-                    });
+                prometheus = scope.Pointer(options.Prometheus.ToInteropOptions(scope));
```

Comment:
This shows that nobody was using advanced Prometheus options beyond just the address since we weren't converting properly anyways